BUILDING.md

```diff
@@ -44,14 +44,25 @@ $ cd butter
 
 To install Butter into `/usr`:
 
-```
-$ meson --prefix=/usr build
+```sh
+$ meson setup --prefix=/usr build
 $ ninja -C build
 # ninja -C build install
 ```
 
 To uninstall:
 
-```
+```sh
 # ninja -C build uninstall
 ```
+
+## Updating Translations
+
+- The below steps are recommended to be done maintainers to avoid conflicts with any open PRs.
```

Comment:
This step is for you and other maintainers, not for translators to prevent merge conflicts. I will remove the update PO line as those files shouldn't be modified even by maintainers and must instead be synced by translators in their programs like Poedit.